todo_app/app.py

```diff
@@ -8,4 +10,17 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    
+    return render_template('index.html', items = items)
+
+
+@app.route('/item', methods=['POST']) 
+def add_item_to_list():
+
+    itemTitle = request.form.get('item-input')
+    if itemTitle != "":
```

Comment:
minor, and although this is good! It's a better idea to put a `required` attribute on the input in the frontend, that way a user won't even be able to submit an empty title

Reply:
Thanks very much - have fixed in commit a943886